common/datavalues/src/types/type_timestamp.rs

```diff
@@ -106,9 +123,6 @@ impl DataType for TimestampType {
             4 => &["DateTime(4)"],
             5 => &["DateTime(5)"],
             6 => &["Timestamp", "DateTime"],
-            7 => &["DateTime(7)"],
-            8 => &["DateTime(8)"],
-            9 => &["DateTime(9)"],
```

Comment:
Why we don't support precision 9?

Reply:
for two reasons, one is cockroachDB and duckdb only support timestamp with precision microsecond.
another is that is our timestamp bound of ts is `9999-12-31 23:59:59` whose nanosecond is beyond i64::MAX and to make it simple, we can store the total timestamp in a `i64`

Reply:
ok